portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,115 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+
+    <body>
+
+        <div id="content">
+            <h1 class="center">Ashish Agarwal</h1>
+            <p class="center">ashish01012000@gmail.com | 844.782.2348 | ashis170123011@iitg.ac.in</p>
+            <p>I'm a final year B.Tech(Mathematics and Computing) student at IIT-Guwahati.
+                I spend most of my free time in competitive programming and watching animes, web series.
```

Comment:
You should suggest me more web series regarding history, strategy and economics.

Reply:
House of Cards is a really good combination strategy and politics. I don't know much about history or economics. 
I am more of an Anime guy. Deathnote is definitely worth watching if you like strategy.

Reply:
Sure, thanks.